src/client.js

```diff
@@ -391,6 +391,8 @@ Client.prototype.open = function(socketId, data) {
 };
 
 Client.prototype.sort = function(data) {
+	self.emit("sync_sort", data); // Sync order to connected clients
```

Comment:
This should be done along with save call at the bottom. Also I noticed it will still save even if you sort a non existing channel.